scripts/importData/index.js

```diff
@@ -9,6 +9,8 @@ const csvConfig = {
     'courseCode',
     'courseDisposition_sv',
     'courseDisposition_en',
+    'recommendedPrerequisites_sv',
+    'recommendedPrerequisites_en',
```

Comment:
I'm not sure, but I think we need to create a new import script for just recommendedPrerequisites_sv/en. But for now I think we can wait with that, ie not updating importData/index.js and importData/index2.js at all